tests/test-tag-and-attribute-sanitizer.php

```diff
@@ -1203,7 +1215,7 @@ public function test_replace_node_with_children_validation_errors() {
 		);
 		$expected_errors[] = array(
 			'node_name'       => 'foo',
-			'parent_name'     => 'body',
+			'parent_name'     => 'divs',
```

Comment:
It seems like the `parent_name` should be `divs`, as that's the parent of `<foo>`. But feel free to offer a different opinion.

Reply:
You're right. The same change was required in https://github.com/ampproject/amp-wp/pull/3243/commits/f084b43156c89856ef8dd0f2dcb8e34d20fe71f0 for #3243.